cmd/app/internal/service/tts-stt/tts-stt.py

```
@@ -91,50 +142,176 @@ async def speech_to_text(file: UploadFile):
             os.unlink(tmp_path)
             logger.debug(f"Deleted temp file {tmp_path}")
 
-
 # -----------------------------------
 # Text to Speech
 # -----------------------------------
+def generate_audio_chunk(text_chunk: str, output_path: str):
+    """Generate audio for a text chunk"""
+    try:
+        tts_fast.tts_to_file(text=text_chunk, file_path=output_path)
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)